Source/WebCore/css/mathml.css

```diff
@@ -143,12 +144,10 @@ math[display="block" i] {
     page-break-inside: avoid;
     math-style: normal;
 }
-math[display="inline" i] {
-  math-style: compact;
-}
 
-ms, mspace, mtext, mi, mn, mo, mrow, mfenced, mfrac, msub, msup, msubsup, mmultiscripts, mprescripts, none, munder, mover, munderover, msqrt, mroot, merror, mphantom, mstyle, menclose, semantics, mpadded, maction {
-    display: block;
+math[display="inline" i] {
+    display: inline;
```

Comment:
Ditto.

Reply:
Done in https://github.com/WebKit/WebKit/commit/b34d57188d64ad705feb68336e9367605315401e.